bin/version_check

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+# USAGE: ./bin/version_check || circleci step halt
+
+changes=`git show -m HEAD --name-only --pretty=""`
```

Comment:
I need to be reminded - what does `git show` show? Is it only the last commit? What if a PR has multiple commits?

Reply:
> So any changes to `.json` files or any files in `/src/schema` will result in a circleci failure - am I reading this properly?

Right, this is just the first step, making sure the bash script can fail CircleCI.

Next step is to only fail if there are those changes and package.json isn't also changes.

Then I want to actually check that the version string of package.json is updated.

The stretch goal would be to check that the version string is *incremented* in package.json. But I think the tool is still useful if we stop at the immediate previous step.

Reply:
Yeah, it'll be `git diff origin/master` in the next iteration.